### PR TITLE
Update docs (2)

### PR DESCRIPTION
**Pull request by ptobuon**

### CODE REVIEW

**Review by MishaTY — DISMISSED** [review]

**Comment by quajak** [comment]
Why delete the readme-it.md?

File: Build/Test Kernel/readme.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ nasm -f bin -o output.bin kernel.asm
 build iso:
 mkisofs -R -b isolinux.bin -no-emul-boot -boot-load-size 4 -boot-info-table -o boot.iso
 
-then use boot.iso as boot image.
\ No newline at end of file
+then use boot.iso as boot image.
```
Comment (ascpixi) [inline Build/Test Kernel/readme.md:10]: i think we can format this file while we're at it :P

File: Build/readme.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Bin
```
Comment (ascpixi) [inline Build/readme.md:1]: i think changing these to headers would look nice!

File: ReadMe - Solutions.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+# Solution File ReadMe
```
Comment (ascpixi) [inline ReadMe - Solutions.md:1]: i feel like renaming this file to `SOLUTIONS.md` would work better for consistency, as we have files like `README.md`, `CODE_OF_CONDUCT.md`, `LICENSE.txt` and `CONTRIBUTING.md`

**Review by quajak — APPROVED** [review]
Thanks for updating and cleaning up the documentation!

**Pull request merged**